src/Adapter/Module/Tab/ModuleTabRegister.php

```diff
@@ -266,4 +266,56 @@ protected function registerTab(Module $module, ParameterBag $data)
                     'Admin.Modules.Notification'));
         }
     }
+
+    /**
+     * Find the parent ID from the given tab context
+     *
+     * @param ParameterBag $data The structure of the tab.
+     * @return int ID of the parent, 0 if none
+     */
+    protected function findParentId(ParameterBag $data)
+    {
+        $idParent = 0;
+        $parentClassName = $data->get('parent_class_name', $data->get('ParentClassName'));
+        if (!empty($parentClassName)) {
+            // Could be a previously duicated tab
+            $idParent = (int)$this->tabRepository->findOneIdByClassName($parentClassName.self::SUFFIX);
```

Comment:
maybe only cast the last call of `$idParent` in duplicateParentIfAlone function?